README.md

```diff
@@ -52,11 +52,12 @@ The version of the extension for the Opera browser can be found at the [Opera ad
 To get a local copy of the current code, clone it using git:
 
     $ git clone git://github.com/mozilla/pdf.js.git pdfjs
-    $ cd pdfjs
+    $ cd pdf.js
```

Comment:
or maybe change the actual clone command to use pdf.js ? then its the same url you would get as when using github tools to clone.